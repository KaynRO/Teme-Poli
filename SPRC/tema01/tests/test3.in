login kayn
load
read 1
logout
login andrei
add 1 3 1.2 2.2 3.3
store
read 1
logout
login kayn
load
read 1
logout
