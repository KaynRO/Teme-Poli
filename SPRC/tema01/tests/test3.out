--------------------------------
Session key: 1605611193
--------------------------------
Load successfull
--------------------------------
Data ID does not exists
--------------------------------
Logout successfully
--------------------------------
Session key: 1605611193
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
No database loaded into memory
--------------------------------
Logout successfully
--------------------------------
Session key: 1605611193
--------------------------------
Load successfull
--------------------------------
Data ID does not exists
--------------------------------
Logout successfully
